spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -58,6 +58,24 @@
         expect(group_assignment.group_assignment_repos.count).to eql(0)
         expect(user.repo_accesses.count).to eql(0)
       end
+
+      it 'does not allow user to join if group has reached maximum number of members' do
```

Comment:
You should include the test setup in the `context` and the actual test run in the `it`